packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts

```diff
@@ -1115,6 +1149,10 @@ const getEchartAxes = ({
                         : showGridX,
                 },
                 inverse: !!xAxisConfiguration?.[0].inverse,
+                data: useWeekAxisConfiguration ? customXRange : undefined,
+                axisTick: useWeekAxisConfiguration
+                    ? { alignWithLabel: true, interval: 0 }
+                    : undefined,
```

Comment:
Curious about this change as ticks seem to be used as delimiters for categorical axis. So should we change this for all categorical axis or for none or just for weeks 🤔 
Example 

<img width="971" alt="Screenshot 2024-05-03 at 10 07 49" src="https://github.com/lightdash/lightdash/assets/9117144/61eea979-0a5c-49a7-98e8-13e30aa87cf7">

Reply:
I think just for weeks. We want weeks to behave more like a time axis, but we have to use category axis to get there. In the time axes, the ticks line up with the data and labels.